pkgs/development/libraries/openldap/default.nix

```diff
@@ -13,12 +15,17 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  buildInputs = [ openssl cyrus_sasl db groff ];
+  buildInputs =
+       [ openssl cyrus_sasl db groff ]
+    ++ stdenv.lib.optional enableModules libtool
+    ;
 
   configureFlags =
     [ "--enable-overlays"
       "--disable-dependency-tracking"   # speeds up one-time build
-    ] ++ stdenv.lib.optional (openssl == null) "--without-tls"
+    ]
+      ++ stdenv.lib.optional enableModules [ "--enable-modules" "LDFLAGS=-L$out/lib" ]
```

Comment:
Yes, but I don't think it will be used, if you put it here.
I think it was not necessary for me.